src/appReview/selectors.ts

```diff
@@ -0,0 +1,3 @@
+import { RootState } from 'src/redux/reducers'
+
+export const appReviewSelector = (state: RootState) => state.appReview
```

Comment:
why do we not have more specific selectors? components using `appReviewSelector` now will receive an update any time anything in this whole `appReview` object, which seems like a weird thing to want. it also doesn't fit the pattern we have in other selector files?